index.html

```diff
@@ -118,6 +123,17 @@
             "!WEBIDL#securityerror"><code>SecurityError</code></a>, and abort
             these steps.
             </li>
+            <li>Let <var>url</var> be the result of running the <a data-cite=
```

Comment:
TypeError sounds good. It matches [fetch](https://fetch.spec.whatwg.org/#request-class).